workers/unity/Packages/io.improbable.gdk.core/Utility/Snapshot.cs

```diff
@@ -21,6 +21,11 @@ public class Snapshot
         /// <returns>The next available entity ID.</returns>
         public EntityId GetNextEntityId()
         {
+            while (!IsValid(new EntityId(nextEntityId)))
+            {
+                nextEntityId++;
+            }
```

Comment:
I can make this O(log N) but it would need an array of `EntityId`. I didn't think it would matter here. Thoughts?

Reply:
Yeah this is fine I think.